.babelrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please use 2 spaces like in other config files (e.g. webpack, tsconfig, eslintrc).